recipes/gdal2/meta.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: gdal
+  version: 2.1.0.trunk
+  
+source:
+  svn_url: https://svn.osgeo.org/gdal/trunk/gdal
```

Comment:
I would be interested in hooking up a **latest source** branch in conda-forge. (Even if it is just for testing the build; no uploads to the channel.)